playbooks/install_os.yml

```diff
@@ -4,6 +4,17 @@
   hosts: all
   strategy: linear
   become: true
+  vars:
+    default_timesync_ntp_servers:
+      - '0.pool.ntp.org'
+      - '1.pool.ntp.org'
+      - '2.pool.ntp.org'
+      - '3.pool.ntp.org'
```

Comment:
I don't think EDPM should maintain a list of default NTP servers. We don't want to repeat what we did with tripleo-heat-templates, where we had to maintain defaults for components on our own. In this case, the timesync role does not maintain defaults, so EDPM shouldn't either. Instead, we can rely on the defaults set by the distro.

Reply:
Ok. That does make sense.